src/integrationtests/resources/com/aws/greengrass/integrationtests/metrics/metricsConfig.yaml

```diff
@@ -0,0 +1,104 @@
+---
+services:
+  main:
+    lifecycle:
+      install:
+        all: echo All installed
+    dependencies:
+      - aws.greengrass.clientdevices.Auth
+      - BrokerSubscribingToCertUpdates
+      - BrokerWithGetClientDeviceAuthTokenPermission
+      - BrokerWithAuthorizeClientDeviceActionPermission
+  BrokerSubscribingToCertUpdates:
+    dependencies:
+      - aws.greengrass.clientdevices.Auth
+    lifecycle:
+      run:
+        windows:
+          powershell -command sleep 1
+        posix:
+          sleep 1
+    configuration:
+      accessControl:
+        aws.greengrass.clientdevices.Auth:
+          policyId1:
+            policyDescription: access to certificate updates
+            operations:
+              - '*'
+            resources:
+              - '*'
+        aws.greengrass.ipc.pubsub:
+          policyId2:
+            policyDescription: access to pubsub topics for ServiceName
+            operations:
+              - '*'
+            resources:
+              - '*'
+  BrokerWithGetClientDeviceAuthTokenPermission:
```

Comment:
what are all these components for? They don't appear to be used at all. Your test is manually emitting metrics.

Reply:
These are for tests that trigger the different APIs as well, I left them out of the PR right now to keep it short, but I'll add those tests in next